lib/LocalVerifiableCredentialStore.js

```diff
@@ -131,6 +139,26 @@ export class LocalVerifiableCredentialStore {
     }
   }
 
+  /**
+   * Finds the best matching verifiable credential for the given query.
+   *
+   * @param {object} options - The options to use.
+   * @param {object} options.query - The query used to find the match.
+   *
+   * @returns {Promise<Array>} List of matching VCs.
+   */
+  async match({query}) {
+    const {type} = query;
```

Comment:
It would be good to do some validation on `query` here to ensure it is not `undefined`.